nest-forms-backend/src/files/files.helper.ts

```diff
@@ -367,4 +369,26 @@ export default class FilesHelper {
     })
     return uuids
   }
+
+  /*
+  Sends request to nest-clamav-scanner to delete file from scanner database, so the file will not be scanned
+   */
+  async deleteFileFromScannerClient(
+    scannerId: string,
+  ): Promise<GetScanFileDto | undefined> {
+    this.logger.log(
+      `Deleting file with scannerId: ${scannerId} from scanner...`,
+    )
+
+    const responseScanner =
+      await this.scannerClientService.deleteFile(scannerId)
```

Comment:
Since this function can throw error ([see here](https://github.com/bratislava/konto.bratislava.sk/blob/f0959942dd3c7360804d6337d13c85e0b37069f8/nest-forms-backend/src/scanner-client/scanner-client.service.ts#L160)), shouldn't this be in trycatch and if it fails, it should then throw the new `FILE_DELETE_FROM_SCANNER_ERROR` error? Because if the request fails we will see just some generic AxiosError no?

Reply:
I used a same patter which we have on scanner client the functions like here on `notifyScannerClient` where basic try catch is done inside `this.scannerClientService.scanFile` and helper is mostly used for logging and calling the scanner client service. Then errors are handled in level above in `files.service.ts` like here
```

 const scannerResponse = await this.filesHelper.notifyScannerClient(
      minioFileName,
      formInfo,
    )

    if (!scannerResponse) {
      throw this.throwerErrorGuard.InternalServerErrorException(
        FilesErrorsEnum.SCANNER_NO_RESPONSE_ERROR,
        FilesErrorsResponseEnum.SCANNER_NO_RESPONSE_ERROR,
      )
    }
```

I would stick with it. If we want to change it (I am free to discuss it), then we have to do it in all scanner client methods, then we have a same approach everywhere.

Reply:
okay, seems fair